lib/plugins/aws/customResources/resources/utils.test.js

```diff
@@ -1,6 +1,5 @@
 'use strict';
 
-// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
Same with this one

Reply:
will undo this as this was change after adding `.eslintrc.js`